ThirdPartyModules/Mirasvit/Credit.php

```diff
@@ -183,4 +191,56 @@ public function checkMirasvitCreditAdminQuoteUsed($result, Observer $observer)
 
         return false;
     }
+    
+    /**
+     * To run filter to check if the Mirasvit credit amount can be applied to shipping/tax.
+     *
+     * @param boolean $result
+     * @param Mirasvit\Credit\Api\Config\CalculationConfigInterface|Mirasvit\Credit\Service\Config\CalculationConfig $mirasvitStoreCreditCalculationConfig
+     * @param Quote|object $quote
+     * 
+     * @return boolean
+     */
+    public function checkMirasvitCreditIsShippingTaxIncluded($result,
+                                     $mirasvitStoreCreditCalculationConfig,
+                                     $quote)
+    {
+        $this->mirasvitStoreCreditCalculationConfig = $mirasvitStoreCreditCalculationConfig;
+        
+        return $this->ifMirasvitCreditIsShippingTaxIncluded($quote);
+    }
+    
+    /**
+     * If the Mirasvit credit amount can be applied to shipping/tax.
+     *
+     * @param Quote|object $quote
+     * 
+     * @return boolean
+     */
+    private function ifMirasvitCreditIsShippingTaxIncluded($quote)
+    {   
+        if ($this->mirasvitStoreCreditCalculationConfig->isTaxIncluded($quote->getStore()) || $this->mirasvitStoreCreditCalculationConfig->IsShippingIncluded($quote->getStore())) {
```

Comment:
@dumega Can we refactor the code like this?
return $this->mirasvitStoreCreditCalculationConfig->isTaxIncluded($quote->getStore()) || $this->mirasvitStoreCreditCalculationConfig->IsShippingIncluded($quote->getStore());